src/System.Runtime.CompilerServices.VisualC/tests/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap-Windows_NT;
```

Comment:
I pulled your changes down locally and updated this configuration to just have netstandard and then did a build and run of the tests on netfx and it worked for me. The assembly version for the test library was also 4.1.1.0 as I would expect.